moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -370,7 +402,7 @@ const NewProjectMap = ({
               key={config.layerIdName}
               type="vector"
               tiles={[config.layerUrl]}
-              maxzoom={config.layerMaxLOD || mapConfig.mapboxDefaultMaxZoom} // maxLOD found in vector tile layer metadata
+              maxZoom={config.layerMaxLOD || mapConfig.mapboxDefaultMaxZoom} // maxLOD found in vector tile layer metadata
```

Comment:
i noticed this typo in the maxZoom prop name. did not have any effect on UI as far as i can tell.